docs/getting-started.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_position: 2
+sidebar_label: Getting Started (High-Availability Clusters)
+title: "Getting Started (High-Availability Clusters)"
```

Comment:
@albinsun I updated the title because I plan to create another Getting Started topic for single-node clusters, as you suggested.

Reply:
Hmm... do you mean 2 differnt **Getting Started** pages?
I think your current structure w/ one **Getting Started** page is great and we should keep it.

What I mean is we can have someway to
1. Let user clearly know this step is for multi- or single-node. (e.g. via mark `(Optional)`)
2. Consider to convert small or optional steps by info block or something
   -> This just my personal preference, we should follow your professional advice on tech doc writing.

Reply:
We need 2 Getting Started pages to make reading easier for users. If they already know that they don't need HA, we shouldn't show them HA-related information. Also, single-node clusters require a special StorageClass with a replica count of 1. I do not want to show that in the HA procedure.

@bk201 Let me know if there are other differences. For example, can single-node users use the management network for their VM network? In this page, we tell them to create a custom cluster network.

Reply:
Hi @jillian-maroket,

From previous discusstion, now I'm with your original design to use HA cluster by default but suggest not to have 2 Getting Started pages. IMO:
1. User generally expect 1 **Getting Started** page, so we can let HA cluster the default as _it's our recommand setting on production_.
2. The **single-codes cluster** in description is sufficient for users who just want a try. And I suggest not to have an individual page cuz we don't want user use single node cluster for production.

![image](https://github.com/harvester/docs/assets/2773781/04ae3731-3689-4c2a-bf46-aca2e0f5ae5f)

However I'm not a professional technical writer and it's just my personal opinion, if we are going to have 2 pages, we may consider to organize them like
![image](https://github.com/harvester/docs/assets/2773781/02d2a88c-56a8-4bfd-b9bb-6677feabeb66)

Thank you.

Reply:
@albinsun I really appreciate your feedback but I don't think I have to justify every single doc-related improvement that I want to implement. Just to wrap up:

> Also, single-node clusters require a special StorageClass with a replica count of 1.
> This is not a necessary config as function still works but just with warning for insufficient replicas.

This is still a concern that is unique to single-node clusters and should not be mentioned in the HA-specific content. If this is an optional step, then I will mark it as optional but I will still mention it.

> User generally expect 1 Getting Started page

What user behavior tracking data or professional documentation experience can back up this very big generalization? FYI the amount of getting started content depends on the complexity of the product.  

> And I suggest not to have an individual page cuz we don't want user use single node cluster for production.

We provide information because we want to improve the overall user experience, not because we are encouraging them to use the product in a specific way. The guide for single-node clusters is for users who don't have the time or resources to deploy HA. We still want them to try Harvester's features so why not make life easier for them?

Again, I appreciate the feedback. I'll create a Getting Started section with separate files for HA clusters and single-node clusters.